src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
Mayokek, i said, it uses a iterator to follow the order of the pool, not random.